apps/dashboard/src/components/project-navigation/index.js

```diff
@@ -53,13 +61,30 @@ const ProjectNavigation = ( {
 				>
 					{ __( 'Editor', 'dashboard' ) }
 				</TabNavigation.Tab>
-				<TabNavigation.Tab
-					isSelected={ activeTab === Tab.RESULTS }
-					isDisabled={ ! projectId }
-					href={ `/project/${ projectId }/results` }
+
+				<span
+					ref={ noteSpan }
+					onMouseEnter={ () => setDisplayNotice( true ) }
+					onMouseLeave={ () => setDisplayNotice( false ) }
 				>
-					{ __( 'Results', 'dashboard' ) }
-				</TabNavigation.Tab>
+					<PopoverNotice
+						context={ noteSpan }
+						onClose={ () => setDisplayNotice( false ) }
+						isVisible={ displayNotice && ! projectId }
+						position={ 'bottom left' }
+					>
+						{ __(
+							'Please save draft or publish project to view results'
```

Comment:
i18n context is missing ("dashboard")

Reply:
Could you explain a little more?  the line `{ __( 'Results', 'dashboard' ) }` was already there.  Should 'dashboard' not be wrapped in the i18n?